### PR TITLE
Armand local

### PR DESCRIPTION
**Pull request by Fogha**
Made a lot of changes to the file structures, deleted some files, did not touch any code. 
Also changed some file names.

### CODE REVIEW

File: src/main/webapp/app/styles/app.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // @import 'node_modules/bootstrap/scss/bootstrap';
 // @import '~bootswatch/dist/materia/bootswatch';
 .bg-image {
-  background-image: url('./bounties_bg.jpg');
+  background-image: url('../assets/bounties_bg.jpg');
```
Comment (xurror) [inline src/main/webapp/app/styles/app.scss:7]: Most media like images, svgs etc... are in a folder named content. So I dont know, which is better, "content" or "assets" but either way it makes more sense if they are all in thesame location. Agreed?

**Review by xurror — CHANGES_REQUESTED** [review]
Don't move the bounties-jdl.png file. It's not one of the UI assets.

**Comment by xurror** [comment]
It's looking good.

Some few things I'd like point out;
 - Anything that is not in */webapp folder is not a UI asset. It's either a backend only asset or a global asset so be wary when interacting with those as more than the UI may end up being affected.
 - Now that there are more people working on this, it will a good practice to always rebase with upstream code before a pull request so we can avoid merge conflicts.
 - So as I pointed in my earlier comment, images and others alike already had a folder named content. So, it's either we move all those to assets folder or we just use content folder but not both.
 - Travis CI will most likely fail too so don't care too much about that now. I will sort it out eventually.

I guess that's all for now. Good work so far.

**Comment by xurror** [comment]
It's looking good.

Some few things I'd like point out;
 - Anything that is not in */webapp folder is not a UI asset. It's either a backend only asset or a global asset so be wary when interacting with those as more than the UI may end up being affected.
 - Now that there are more people working on this, it will a good practice to always rebase with upstream code before a pull request so we can avoid merge conflicts.
 - So as I pointed in my earlier comment, images and others alike already had a folder named content. So, it's either we move all those to assets folder or we just use content folder but not both.

File: src/main/webapp/app/styles/app.scss (hunk 1 above)
Comment (Fogha) [reply src/main/webapp/app/styles/app.scss:7]: Agreed. Here i will put just logos, and images. Since u have trouble loading them from content/images

**Comment by xurror** [comment]
You still have merge conflicts here. Please rebase your codebase with upstream master

File: src/main/webapp/app/styles/app.scss (hunk 1 above)
Comment (xurror) [reply src/main/webapp/app/styles/app.scss:7]: It's not that the app couldn't load from content, I just had challenges referencing the path. But that's ok.

**Review by xurror — APPROVED** [review]
Looks good.
Just rebase with upstream master, resolve the merge conflicts and I can merge this.

**Pull request closed without merging**